packages/gatsby-theme-carbon/src/components/InlineNotification/InlineNotification.js

```diff
@@ -1,41 +1,39 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import classnames from 'classnames';
+import cx from 'classnames';
 import { InlineNotification as CarbonInlineNotification } from 'carbon-components-react';
 import { Row, Column } from '../Grid';
+import useMetadata from '../../util/hooks/useMetadata';
 
-import { notification } from './InlineNotification.module.scss';
+import styles from './InlineNotification.module.scss';
 
-export default class InlineNotification extends React.Component {
-  render() {
-    const { children, className, kind } = this.props;
+const InlineNotification = ({ children, className, kind = 'info' }) => {
+  const notificationClasses = cx(styles.notification, {
+    [className]: className,
+  });
 
-    const notificationClasses = classnames(notification, {
-      [className]: className,
-    });
+  const { interiorTheme } = useMetadata();
 
-    return (
-      <Row>
-        <Column colLg={8} colMd={6} className={notificationClasses}>
-          <CarbonInlineNotification
-            lowContrast
-            hideCloseButton
-            kind={kind}
-            title=""
-            subtitle={children}
-          />
-        </Column>
-      </Row>
-    );
-  }
-}
+  return (
+    <Row>
+      <Column colLg={8} colMd={6} className={notificationClasses}>
+        <CarbonInlineNotification
+          lowContrast
+          hideCloseButton
+          kind={kind}
+          title=""
+          subtitle={children}
+          className={cx({ [styles.darkMode]: interiorTheme === 'dark' })}
```

Comment:
I don't mind this, it's nice if we add more classnames down the line. But just want to make sure you know this works too.

When there's just one class, it's a tad easier to read IMO

```suggestion
          className={interiorTheme === 'dark' && styles.darkMode}
```